pyproject.toml

```diff
@@ -22,14 +22,14 @@ Jinja2 = "^3.0.3"
 importlib-resources = "^5.4.0"
 exasol-bucketfs-utils-python = { git = "https://github.com/exasol/bucketfs-utils-python.git", branch = "main" }
 exasol-data-science-utils-python = {git = "https://github.com/exasol/data-science-utils-python.git", branch = "main"}
+exasol-udf-mock-python = { git = "https://github.com/exasol/udf-mock-python.git", branch = "master" }
```

Comment:
this shouldn't be a real dependency, it should only be an dev dependency, is there a reason